test/e2e/multi_record_test.go

```diff
@@ -824,7 +824,7 @@ var _ = Describe("Multi Record Test", Labels{"multi_record"}, func() {
 					err := tr.cluster.k8sClient.Get(ctx, client.ObjectKeyFromObject(tr.record), tr.record)
 					g.Expect(err).To(MatchError(ContainSubstring("not found")))
 				}
-			}, recordsRemovedMaxDuration, 5*time.Second, ctx).Should(Succeed())
+			}, GetRecordsDeletedTimeout(testDNSProvider), 5*time.Second, ctx).Should(Succeed())
 			GinkgoWriter.Printf("[debug] all records removed in %v\n", time.Since(checkStarted))
 
 			By("checking provider zone records are all removed")
```

Comment:
Rather than calling the function all over the place you might be better to just set the values at the start [here](https://github.com/Kuadrant/dns-operator/blob/f7aa99474d84888eb6bf45cad229b996b1dd5cb2/test/e2e/multi_record_test.go#L45C2-L45C27) 

```
recordsReadyMaxDuration := GetRecordsReadyTimeout(testDNSProvider)
recordsRemovedMaxDuration := GetRecordsDeletedTimeout(testDNSProvider)
```

All the [log statements](https://github.com/Kuadrant/dns-operator/blob/f7aa99474d84888eb6bf45cad229b996b1dd5cb2/test/e2e/multi_record_test.go#L130) would be printing the correct thing that way.

Reply:
yeah, that's neater. I've updated it now.